plugins.json

```diff
@@ -2422,6 +2422,58 @@
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-incident-management-v1.1.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-incident-response/releases/tag/v1.1.0",
+    "hosting": "",
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl+gbc0ACgkQ0bVLR6XO/sRB+g//fo8T73j8PdYHwM9wj6glUBnwQBIeV/tuLjWr7e7RhuG8p0aeF5vErZ0bFQT+aDKGlF7E6s/tok3PvouthwjY8pOxgr/R0ReKH93ou1ivVlicn+mglMKIou89ViO1yMhjt2l/sc6pc0muyaHjwmfs9M1I2/nJ0qWkcTP7Dvx8E/EFiXaUYww8TNGqf89FbStdthgg27mVa1xg7J79TRODjCl5W+or+nD51RJzh1dhC+FPWTyGWTNJQ0KaYwOI7txOc0qbOJiDKwX9rbcNG7X1AK6A9GT07gS6IW5c5/ajQNnNJeqdRtbUdeMVk6DK6yEmvKhLUByBR4R3d8FXNUyko5k7QNQF4F4ozn76CuL7Z+JCi0IOWo4H1u+4wRm+kf7TYcOFovCyXsIw8wzcbrGI8+QyeQ2qJuOEk1BcMiMU27s7Hkrm1uQ/F0IlbtKR0wh3sX+tFW5hc4iqMjFSHcy7bsAaIEJ64JdZgrdhrTTS9HXQ5LhtYy+Ar5y5J26WHJ5yjbXcrkuZQ2MAo7marE8RbSPQpmc8S/EEdjm2M37vpf0IHUoMTpgmebDbJMgl1whoKLRKFIpM0+nbDjQKdmInrSUCGQ85J9uDs3wSstVQB1t2ntNEa5kZ9Pv8noVi/NwbsGCJEIJucKyMSySE6gP4ugvtNzFmZg9WZEz/fB6R6Ec=",
+    "repo_name": "mattermost-plugin-incident-management",
+    "manifest": {
+      "id": "com.mattermost.plugin-incident-management",
+      "name": "Incident Management",
+      "description": "This plugin allows users to coordinate and manage incidents within Mattermost.",
+      "homepage_url": "https://github.com/mattermost/mattermost-plugin-incident-response/",
+      "support_url": "https://github.com/mattermost/mattermost-plugin-incident-response/issues",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-incident-response/releases/tag/v1.1.0",
+      "icon_path": "assets/incident_plugin_icon.svg",
+      "version": "1.1.0",
+      "min_server_version": "5.28.0",
+      "server": {
+        "executables": {
+          "linux-amd64": "server/dist/plugin-linux-amd64",
+          "darwin-amd64": "server/dist/plugin-darwin-amd64",
+          "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+        },
+        "executable": ""
+      },
+      "webapp": {
+        "bundle_path": "webapp/dist/main.js"
+      },
+      "settings_schema": {
+        "header": "",
+        "footer": "",
+        "settings": []
+      }
+    },
+    "enterprise": false,
```

Comment:
I think we need to use the `--enterprise` flag so that this gets set to true and only gets shown for E20 customers.

Reply:
done!